test-case/volume-basic-test.sh

```diff
@@ -58,7 +58,7 @@ sleep 1
 [[ ! $(pidof aplay) ]] && die "$pid process is terminated too early"
 
 sofcard=${SOFCARD:-0}
-pgalist=($(amixer -c"$sofcard" controls | grep -i PGA | sed 's/ /_/g;' | awk -Fname= '{print $2}'))
+pgalist=($(amixer -c"$sofcard" controls | grep "[0-9]\.[0-9]" | sed 's/ /_/g;' | awk -Fname= '{print $2}'))
```

Comment:
I am not sure if we have other controls takes this form in IPC4, if no, it is good.
another possible fix can can be `grep -E PGA|gain`

Reply:
BTW this code was overly complicated. It can be simplified like this:
```shell
pgalist=($(amixer -c"$sofcard" controls | awk -Fname= '/PGA/ { print $2 }')
```
With `-Fname` there's no reason to convert spaces back and forth.